### ISSUE TITLE
Internal Server Error

### ISSUE THREAD

**Issue by deepanshumehtaa** [PadmavathySundaresan/Diabetic_Retinopathy#2]
Hello, i am facing this error, because you did't link your `html.htlm` with your flask application.
Please correct it !!

**Comment by PadmavathySundaresan** [comment]
Yes, Please add the name of html file as an extension in the ip address. So that you can view the html page.

**Comment by deepanshumehtaa** [comment]
and how can i add this??, please tell me..

**Comment by deepanshumehtaa** [comment]
i have done this, but still it is not working.....
